docs/data-model.md

```diff
@@ -0,0 +1,76 @@
+# Data model
+
+> [!NOTE]
+> This is a _draft_ document. It will change as we develop our understanding of the services that this application encapsulates.
+
+This application will power two user-facing services: 'Manage school placements' and 'Claim funding for mentors' (a.k.a. Track & Pay). Some data entities will be used by both services – for example, Mentors, Providers and Schools. Others will only be relevant to one service – for example, Funding Claims and School Placements.
+
+## Entity Relationship Diagram (ERD)
+
+This diagram represents our current understanding of the data models that will exist within this application.
+
+There are a few things to bear in mind when reading this:
+
+- This diagram attempts to bridge the gap between a 'high level' list of entities, and a 'low level' database schema. It sits somewhere in between.
+- Attribute names have been written in `Pascal_Case_With_Underscores`. This is to aid readability because spaces are not allowed in Mermaid JS ERD attribute names. It does not represent the format of eventual database column names.
```

Comment:
I would rather use rails defaults for naming conventions `Placement_ID`-> `placement_id`. 
`Placement{Placement_ID ...}` feels like it should be `Placement{id}`.

Is the relationship between school and claim not a many to many? Seems like school would be making a claim every year.
Can mentors work for different schools? If so it's a many to many relationship

Reply:
I think Ollie is right that a Claim will only have one school, but a school can have claims. Even if the mentor moves to a different school, the claim is for to compensate the school for the time they lost when the mentor was in training. Only the one school is relevant to the claim. 

From a user perspective, the school user is the person filling in this form, making a claim on behalf of the school. They won't be making a claim for many schools.